build.xml

```diff
@@ -22,8 +22,20 @@
 		<get src="https://nightly.tlapl.us/dist/tla2tools.jar" dest="${lib}"/>
 	</target>
 
-	<target name="compile" depends="download" description="compile the java module overwrites">
-		<javac srcdir="${src}" destdir="${build}" classpath="${lib}/tla2tools.jar"
+	<target name="download-jackson-core" depends="init" description="downloads jackson-core-2.10.2.jar" unless="skip.download">
```

Comment:
Consider adding a directory with `jackson-*.jar` to the git repo to be independent of maven.org.

Reply:
@lemmy where should I put it? Add a `lib` directory?

Reply:
The `lib` directory is ignored